src/Exception/Base/DomainException.php

```diff
@@ -23,9 +23,7 @@ class DomainException extends SplDomainException
     private $errors;
 
     /**
-     * @param string $token
      * @param int $statusCode
```

Comment:
Why do we need to keep `$statusCode` here when it's in the method signature and we have no extra description for this parameter?

Reply:
@nesl247 I haven't reviewed everything, the docblocks that were removed were removed because of lint.

Reply:
@nesl247 https://github.com/LinioIT/common-expressive/pull/17/commits/7814554de4a9fb8ac83b529b5fe23c8cfe902b2d